helix-core/src/syntax.rs

```diff
@@ -154,6 +154,8 @@ pub struct LanguageConfiguration {
     /// Hardcoded LSP root directories relative to the workspace root, like `examples` or `tools/fuzz`.
     /// Falling back to the current working directory if none are configured.
     pub workspace_lsp_roots: Option<Vec<PathBuf>>,
+    #[serde(default)]
+    pub persistent_diagnostic_sources: Vec<String>,
```

Comment:
Could you add a blurb about this to book/src/languages.md?

Reply:
sure sorry forgot about that :smile: I added an entry to the docs